javac/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BASE_IMAGE=openjdk:8-jdk
+ARG BASE_IMAGE=gcr.io/google-appengine/openjdk:8
```

Comment:
I would suggest using the mirror at `launcher.gcr.io/google/openjdk8` because otherwise it might raise questions of why "appengine"?

@dlorenc FYI

Reply:
+1 to using the launcher tag